ynh_composer/ynh_composer__2

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Execute a command with Composer
+#
+# usage: ynh_composer_exec --phpversion=phpversion [--workdir=$final_path] --commands="commands"
+# | arg: -w, --workdir - The directory from where the command will be executed. Default $final_path.
+# | arg: -c, --commands - Commands to execute.
+ynh_composer_exec () {
+	# Declare an array to define the options of this helper.
+	local legacy_args=vwc
+	declare -Ar args_array=( [v]=phpversion= [w]=workdir= [c]=commands= )
+	local phpversion
```

Comment:
phpversion should have a default value, to not have to set at php7.0

Reply:
done but not 100% sure it's the good way to define a default value for an argument